_posts/2024-11-06-#30239.md

```diff
@@ -10,18 +10,38 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Bitcoin Core considers a transaction output to be [dust](https://bitcoin.stackexchange.com/questions/10986/what-is-meant-by-bitcoin-dust) when its value is lower than the cost of creating and spending it. These outputs may be useful in particular circumstances, e.g. L2s, when the parent transaction cannot have fees for technical or accounting reasons.
 
+- Recommend reading from earlier review clubs:
+  - [#30352: Add PayToAnchor(P2A), OP 1 <0x4e73> as standard output script for spending](https://bitcoincore.reviews/30352)
 
 ## Questions
 
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
 
+1. Where in code is it determined if an amount is dust?
+
+1. Is dust restricted by consensus? Policy? Both?
+
+1. How can dust be problematic?
+
+1. What is _ephemeral_ dust?
+
+1. What are the specific proposed rules for ephemeral dust?
+
+1. Why is it important to impose a fee restriction?
```

Comment:
To make it more open-ended, could add follow-ups like:

"Which specific vulnerabilities can you imagine without the fee restriction? Can you think of (in your view) positive use cases that are not possible because of the fee restriction?"

Reply:
Nice, added.